prow/plank/controller.go

```diff
@@ -295,19 +282,7 @@ func (c *Controller) terminateDupes(pjs []prowapi.ProwJob, pm map[string]coreapi
 				}
 			}
 		}
-		toCancel.SetComplete()
-		prevState := toCancel.Status.State
-		toCancel.Status.State = prowapi.AbortedState
-		c.log.WithFields(pjutil.ProwJobFields(&toCancel)).
-			WithField("from", prevState).
-			WithField("to", toCancel.Status.State).Info("Transitioning states.")
-		npj, err := c.kc.ReplaceProwJob(toCancel.ObjectMeta.Name, toCancel)
```

Comment:
On the other hand, currently we will:
* Identify an old job
* delete the pod, ignoring errors
* attempt to mark the job as cancelled